op-node/rollup/derive/params.go

```diff
@@ -30,14 +30,22 @@ const DerivationVersion1 = plasma.TxDataVersion1
 const MaxSpanBatchSize = MaxRLPBytesPerChannel
 
 // MaxChannelBankSize is the amount of memory space, in number of bytes,
-// till the bank is pruned by removing channels,
-// starting with the oldest channel.
+// till the bank is pruned by removing channels, starting with the oldest channel.
 const MaxChannelBankSize = 100_000_000
 
+// MaxChannelBankSizeV1 is the amount of memory space, in number of bytes,
+// till the bank is pruned by removing channels, starting with the oldest channel.
+// This limit is applied after Fjord
+const MaxChannelBankSizeV1 = 1_000_000_000
+
 // MaxRLPBytesPerChannel is the maximum amount of bytes that will be read from
 // a channel. This limit is set when decoding the RLP.
 const MaxRLPBytesPerChannel = 10_000_000
```

Comment:
There are more references to this const, e.g. in span channel out and `op-e2e/actions/garbage_channel_out.go`. We may want to replace all references.